sign/sign.go

```diff
@@ -141,8 +142,25 @@ func (s *Signer) SignImage(reference string) (*SignedObject, error) {
 	}
 
 	if !s.options.AttachSignature {
-		// TODO: https://github.com/kubernetes-sigs/release-sdk/issues/37
-		return &SignedObject{}, nil
+		// We only pass one image at time for now
+		ref, err := s.impl.ParseReference(reference)
+		if err != nil {
+			return &SignedObject{}, fmt.Errorf("parsing reference: %s: %w", reference, err)
+		}
+
+		dig, err := s.impl.Digest(ref.String())
+		if err != nil {
+			return &SignedObject{}, fmt.Errorf("getting the reference digest for %s: %w", reference, err)
+		}
+
+		sigParsed := strings.ReplaceAll(dig, "sha256:", "sha256-")
+		obj := &SignedObject{
+			digest:    dig,
+			reference: ref.String(),
+			signature: fmt.Sprintf("%s:%s.sig", ref.Context().Name(), sigParsed),
+		}
+
+		return obj, nil
```

Comment:
I think this should be done by `VerifyImage`, which denn will be called implicitly by `VerifyImageInternal`.

Reply:
updated